### PR TITLE
fix: publish .map files to npm

### LINKED ISSUE #11: 'Could not read source map' while debugging in VS Code
```
Could not read source map for file:///-/node_modules/lit-xml/dist/src/index.js: ENOENT: no such file or directory, open '-\node_modules\lit-xml\dist\src\index.js.map'
```

### PR DESCRIPTION
**Pull request by hugo-vrijswijk**
Fixes #11

### CODE REVIEW

**Comment by hugo-vrijswijk** [comment]
@nicojs Can you have a look?

File: packages/lit-xml/.npmignore [hunk 1]
```diff
@@ -1,4 +1,5 @@
 **
 !dist/src/**/*
-*.map
+!src/**/*.ts
+tsconfig.json
```
Comment (nicojs) [inline packages/lit-xml/.npmignore:4]: why us this line needed? It should already be excluded with the top level `**`

Comment (hugo-vrijswijk) [reply packages/lit-xml/.npmignore:4]: Yeah I'm not sure to be honest

Comment (hugo-vrijswijk) [reply packages/lit-xml/.npmignore]: Updated it

**Pull request merged**